docs/hugo/content/_index.md

```diff
@@ -23,6 +23,13 @@ This project is stable. We follow the [Kubernetes definition of stable](https://
 
 ASO supports more than 150 different Azure resources, with more added every release. See the full list of [supported resources](https://azure.github.io/azure-service-operator/reference/).
 
+## Contact us
+
+If you've got a question, a problem, an issue, or just want to chat, here are two ways to get in touch.
```

Comment:
```suggestion
If you've got a question, an issue, or just want to chat, here are two ways to get in touch:
```

Reply:
(Feels like problem/issue are the same thing pretty much, and since we're setting our list up I think a `:` to end makes more sense?

Reply:
Rephrased, and using a `:`.